build.gradle

```diff
@@ -42,7 +42,6 @@ subprojects {
   dependencies {
     provided "com.badlogicgames.gdx:gdx:$gdxVersion"
     testCompile "junit:junit:$junitVersion"
-    testCompile "org.mockito:mockito-core:$mockitoVersion"
```

Comment:
Just so you know, `kotlin-mockito` still includes this as compile dependency, so it's not like we're removing it from classpath. I think it's OK to keep it as test dependency, so we can control the included `Mockito` version. Can you revert `build.gradle` and `gradle.properties`?